### ISSUE TITLE
[Feature] update the git clones in the GH actions

### ISSUE THREAD

**Issue by lixuna** [cncf/cnf-testsuite#725]
**Is your feature request related to a problem? Please describe.**
- the test binaries and crystal specs all are failing after the repo name change
- the badge in the README shows failing
- refs #555 

**Describe the solution you'd like**
- update the git clones in the GH actions
- update the Build Status badge svg and url that is displayed at the top of the README.md

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by agentpoyo** [comment]
The build status will get fixed once https://github.com/cncf/cnf-testsuite/pull/742 is approved and merged.

git clones for GH actions should already be addressed and fixed.

@lixuna

**Comment by lixuna** [comment]
looks like the badge is showing "Passing" on https://github.com/cncf/cnf-testsuite

**Comment by lixuna** [comment]
closing as duplicate of #742
